### ISSUE TITLE
Codemagic Windows Build

### ISSUE THREAD

**Issue by AndreiMisiukevich** [rive-app/rive-flutter#394]
### Description
Could you please describe how to build a Windows app with Codemagic?
It seems it's impossible now due to CLang constraint 

### Steps To Reproduce
1. Create a windows app with latest rive package
2. Try to build the app with Codemagic

### Expected behavior
App builds successfully

### Device & Versions (please complete the following information)
- Device: Codemagic Windows machine
- OS: Windows
- Flutter Version: latest

#### Additional context
Related: https://github.com/rive-app/rive-flutter/issues/308

**Comment by AndreiMisiukevich** [comment]
Thanks @eivihnd it works very well
But unfortunately, it increases build time by 4 mins (That is not critical, maybe... but every build now costs extra 0.184$)

**Comment by AndreiMisiukevich** [comment]
@eivihnd do you still use this workaround?
Maybe we can try to convince codemagic to pre-install this?

**Comment by eivihnd** [comment]
@AndreiMisiukevich 
Yes, I still use this on a weekly basis. I agree that it would be great if Codemagic could provide these packages preinstalled.

**Comment by HayesGordon** [comment]
We have drastically changed the way that we build/distribute our native libs from version`0.14.0-dev.1`. Closing this issue as you can use our prebuilt libraries.

You can read more here:
- [Migration Guide](https://rive.app/docs/runtimes/flutter/migration-guide)
- [Rive Native for Flutter](https://rive.app/docs/runtimes/flutter/rive-native)
- [Rive Flutter Docs](https://rive.app/docs/runtimes/flutter/flutter)

If you're having issues updating to the latest, please open a new issue with reproducible steps.
